packages/core/src/parser/StyleParser.test.ts

```diff
@@ -420,8 +420,14 @@ const {
   -- encourage
   A.fn = encourage obj("string1", true, "string\\n", false)
   A.fn2 = encourage obj( a, b )
+  encourage a == b
```

Comment:
Might wanna add some complex expressions for sanity check. Something like:

```suggestion
  encourage a == b
  encourage a + a / b > b*b
  encourage MathPI()*4 > abs(sqrt(b))
```